server/middleware/setCurrentUser.js

```diff
@@ -0,0 +1,5 @@
+module.exports = (req, res, next) => {
+  res.locals.userName = req.user?.getFullName();
+  res.locals.isSignedIn = req.user?.isSignedIn();
```

Comment:
Should the check not be on the existence of a _user_? User is a given class so should always have the methods, but I don't think the extra conditionals hurt it...
```
res.locals.userName = req?.user?.getFullName();
res.locals.isSignedIn = req?.user?.isSignedIn();
  ```

Reply:
Sorry you're right, I'm checking the existence of `req` 😵 I've gone _optional chaining_ slap happy...

Reply:
as discussed